test/modules/home/home_presenter_test.dart

```diff
@@ -171,5 +171,35 @@ void main() {
         verify(view.showUser(user)).called(1);
       });
     });
+
+    describe("it's sideMenuDidShow emits", () {
```

Comment:
```suggestion
    describe("its sideMenuDidShow emits", () {
```
And same for the rest